analyzer/src/main/resources/scripts/init.gradle

```diff
@@ -394,7 +430,7 @@ class AbstractDependencyTreePlugin<T> implements Plugin<T> {
          * @param visited stores the name of already visited dependencies to deal with cycles
          * @return a flag whether the dependency trees are equal
          */
-        private static boolean dependencyTreeEquals(Dependency dependency, DependencyResult dependencyResult,
+        private boolean dependencyTreeEquals(Dependency dependency, DependencyResult dependencyResult,
```

Comment:
Why is this not `static` anymore? It does not seem to need access to `includedBuildsProjects`. Similar below.

Reply:
Yes it does through the call chain `dependencyTreeEquals -> identifierFor(DependencyResult dependencyResult, Project project) -> findRootProjectOfIncludedBuild`.